### ISSUE TITLE
Kubernetes manifests

### ISSUE THREAD

**Issue by Hr46ph** [jsknnr/enshrouded-server#91]
Hey, thanks for your work on this 😬 .

I created a bunch of manifests to deploy a statefulset using your image on my kubernetes cluster, mainly because I wanted to have more control over server configuration. I added a configMap with server settings json and I mount it using subMount for the file inside the pod.

Small changes: only query port udp is required these days, so left out the other port.

If you like I can share the files? I think I can do a fork and submit a merge request or something. Or I can post them here in code tags. Either way, let me know.

**Comment by jsknnr** [comment]
I thought about using a stateful set when I built this out but decided for this use case it may be a little overkill since we only ever have the 1 pod. The config map is a great idea though and something I should have done, but I just leveraged the logic I already built for folks using things like compose and podman. 

Feel free to open a PR and I will take a look.

**Comment by Hr46ph** [comment]
Done!

I think manifests are the easiest way to deploy single pod deployments, nothing overkill about it. Its the docker-compose for Kubernetes, so to speak.

Anyway, have a look!

**Comment by jsknnr** [comment]
Right, I wasn’t referring to the manifests in general, but stateful set vs just a deployment. I’ll have a look when I get some time.
________________________________

**Comment by jogalt** [comment]
Question for either of you, I'm attempting to install via helm and getting the following. @Hr46ph can you clarify how you defined the repo or annotated the index.yaml location?

```
helm install enshrouded steam-servers   --namespace steam-servers   --repo https://github.com/jsknnr/enshrouded-server   --values values.yaml --dry-run
```
```
Error: INSTALLATION FAILED: looks like "https://github.com/jsknnr/enshrouded-server" is not a valid chart repository or cannot be reached: failed to fetch https://github.com/jsknnr/enshrouded-server/index.yaml : 404 Not Found
```

**Comment by jogalt** [comment]
Closing the loop on this. I ended up having to clone the repo locally in rancher/helm gui vs doing a CLI pull. I'm able to stand up the server with no issues now so thanks for your work with packaging this.

As an aside, can you repackage the helm and generate an index.yaml file (for the k8s crowd) and push? This will allow installs without needing to clone the repo. Please and thank you.

```
helm package ./chart-directory
```
```
helm repo index --url https://github.com/jsknnr/enshrouded-server .
```

**Comment by jsknnr** [comment]
This is on my to do list.

**Comment by jsknnr** [comment]
Done. https://jsknnr.github.io/enshrouded-server/
